package.json

```diff
@@ -41,10 +41,12 @@
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^2.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-md5": "^1.2.0",
```

Comment:
Wouldn't this be a `"dependency"`?

Frankly I still find the dependency differences baffling with respect to Ember.

Reply:
Yes good call. I just got that issue when integrating with `profile-engine`. Let's keep this PR open until I've tested in the apps.